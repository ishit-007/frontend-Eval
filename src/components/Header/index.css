.header {
  /* background-color: #8d0085; */
  background-color: black;
  height: 10vh;
  display: flex;
  align-items: center;
  font-family: 'Big Shoulders Display';
}
.header p {
  margin-left: 15%;
  font-size: xx-large;
  color: white;
}
