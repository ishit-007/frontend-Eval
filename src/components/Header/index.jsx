import './index.css';
import React from 'react';

const Header = () => {
  return (
    <div className="header">
      <p>EVENTIFY</p>
      <i className="fa-solid fa-magnifying-glass"></i>
    </div>
  );
};

export default Header;
