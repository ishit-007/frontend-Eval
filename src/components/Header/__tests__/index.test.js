// import { render } from '@testing-library/react';
import Header from '..';
import renderer from 'react-test-renderer';
import React from 'react';

describe('Header', () => {
  it('should render the Header component', async () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
