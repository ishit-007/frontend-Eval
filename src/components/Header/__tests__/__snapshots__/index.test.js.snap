// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the Header component 1`] = `
<div
  className="header"
  style={
    Object {
      "backgroundColor": undefined,
    }
  }
>
  <p>
    EVENTIFY
  </p>
  <i
    className="fa-solid fa-magnifying-glass"
  />
</div>
`;
